CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [2.1.1] - Unreleased
+- Add loaded function to modules, providing an async callback.
```

Comment:
This function should will be into the next feature release  2.2.0 not the 2.1.1 We the follow the Semantic Versioning. Is a new feature not a bug fix.